image/util.go

```diff
@@ -110,3 +117,24 @@ func GetClusterFileFromBaseImage(imageName string) string {
 	}
 	return string(data)
 }
+
+func GetYamlByImage(imageName string) (string, error) {
+	imagesMap, err := imageUtils.GetImageMetadataMap()
+	if err != nil {
+		return "", err
+	}
+
+	image, ok := imagesMap[imageName]
+	if !ok {
+		return "", fmt.Errorf("failed to find image by name")
```

Comment:
why not directly show which image name you input?

fmt.Errorf("failed to find image by name (%s)", imageName)

When returning error or logging, we should provide more and more explicit message as possible.